books/__init__.py

```diff
@@ -10,9 +10,11 @@ def create_app():
     app.config.from_mapping(SECRET_KEY='books')
 
     from . import database
-    from .schema import schema
 
-    app.add_url_rule('/books', view_func=GraphQLView.as_view('graphql', schema=schema, graphiql=True))
+    ### TODO
+    # from .schema import schema
```

Comment:
Why u comment these lines?

Reply:
Because I want to change it will crush now because no queries are specified